src/test/java/org/jitsi/meet/test/ConnectionTimeTest.java

```diff
@@ -346,6 +346,9 @@ private void failIfContainsNull(Double[] array)
         {
             refreshSecondParticipant();
 
+            // just ping the first participant, to avoid timeout of the session
```

Comment:
why don't we increase the timeout ? I remember the need to adjust them for hybrid anyway

Reply:
So this is something else. This timeout is not specific to the test but to the grid. So I was testing and initially, this timeout was very high and if a test crashes the node stay unusable for a long time, so I changed the timeout to 30 seconds. Which means that if a session to a selenium grid node does not receive anything in 30 seconds that session will be closed and the node will be freed. 
In case of this test, we open the first participant and then start reloading the second one for more than a minute (10 reloads). And without touching the first session the timeout is hit and the session expires on the grid. Just executing something fixes the problem, without relying on the value that is on the grid.

Reply:
I know what this is about. The same is with mobile/hybrid when waiting for one mobile to come up other will timeout. There must be a cleaner way, keep alive or something

Reply:
and the case you've described what is it that crashes ? java test process ?

Reply:
one of first google results: http://developers.perfectomobile.com/display/TT/Keeping+long+Selenium+scripts+alive
If we'd have a keep alive thread bound to every participant which dies with java process I guess it would be fine. But given it's a bigger effort I would be fine leaving it for another step with a FIXME here. Eventually I will fix it myself when working on the hybrid

Reply:
I don't see such method in WebDriver interface, seems it is a mobile specific. I don't even see it in the Appium interfaces.
But any request will do the work ...